src/makeCalculator.js

```diff
@@ -38,6 +38,26 @@
  */
 function makeCalculator() {
   // write code here
+
+  const calculator = {
+    result: 0,
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate: (callback, num) => {
+      calculator.result = callback(calculator.result, num);
+
+      return calculator;
```

Comment:
```suggestion
      return this;
```

Reply:
Why doesn`t work when i chanched calculator on this?